_includes/disqus_comments.html

```diff
@@ -0,0 +1,22 @@
+{% if site.disqus_shortname %}
+<script type="text/javascript">
+    /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
+    var disqus_shortname = '{{ site.disqus_shortname }}'; // required: replace example with your forum shortname
```

Comment:
I created a Disqus channel named `ehcache` but it is on my account.
 I would prefer to delete it so that we create the room from another much more official user. do we have one ? Or we use @jhouserizer or @ljacomet account ?